alerts/couchbase/README.md

```diff
@@ -0,0 +1,39 @@
+# Alerts for Couchbase in the Ops Agent
+
+## Spiking Evictions
+
+if ‘bucket.item.ejection.count’ begins to spike by 10 or a user defined threshold it could show that there is unexpected memory pressure.
+
+## Unrecoverable OOM Errors
+
+if `bucket.error.oom.count` that are `unrecoverable` increase above 0 should be a cause for concern
```

Comment:
Please capitalize the first letter in all of the descriptions

Reply:
Could we share why there should be a cause for concern? What does this indicate?